src/main.ts

```diff
@@ -457,34 +457,57 @@ class GeneralSettingsTab extends PluginSettingTab {
         new Setting(this.containerEl)
             .setName("Automatic Task Completion Tracking")
             .setDesc(
-                "If enabled, Dataview will automatically append tasks with their completion date when they are checked in Dataview views."
+                "If enabled, Dataview will automatically append tasks with their completion date when they are checked in Dataview views." +
+                    " Example with default field name and date format: - [x] my task [completion:: 2022-01-01]"
             )
             .addToggle(toggle =>
                 toggle.setValue(this.plugin.settings.taskCompletionTracking).onChange(async value => {
                     await this.plugin.updateSettings({ taskCompletionTracking: value });
+                    // Force reload of settings tab
+                    this.display();
                 })
```

Comment:
I made some comments in the PR about this, with a suggestion to not hide unused options, but to disable them.